src/pokemanki/gui/pokemanki_trade.py

```diff
@@ -32,11 +34,25 @@ def __init__(self, parentObj, mw):
         self.setupUi(parentObj, mw)
 
     def setupUi(self, parent, mw):
+        self.dialog.webEngineView = AnkiWebView(parent=self, title="pokémanki trades")
+        sizePolicy = QtWidgets.QSizePolicy(
+            QtWidgets.QSizePolicy.Policy.Expanding,
+            QtWidgets.QSizePolicy.Policy.Expanding,
+        )
+        sizePolicy.setHorizontalStretch(0)
+        sizePolicy.setVerticalStretch(0)
+        sizePolicy.setHeightForWidth(
+            self.dialog.webEngineView.sizePolicy().hasHeightForWidth()
+        )
+        self.dialog.webEngineView.setSizePolicy(sizePolicy)
+        self.dialog.webEngineView.setObjectName("webEngineView")
+        self.dialog.verticalLayout_2.addWidget(self.dialog.webEngineView)
+
         self.dialog.webEngineView.set_bridge_command(parent.on_bridge_cmd, parent)
 
         # Set in the middle of the screen
         width = 800
-        height = self.height() + 150
+        height = self.height() + 175
```

Comment:
Why set the window here instead of in the designer file?
And what's the reason for setting the height relative to it's original?

Reply:
Because I use this height calculation to control the position on the screen depending on the Anki screen or the monitor size, and for some reason the size also needs to be a bit bigger than whatever QT gives it. If you have any suggestions though, I'm all ears.

Reply:
> whatever QT gives it

Qt gives it the size you set in the Qt Designer:
![Screenshot from 2022-10-24 22-30-30](https://user-images.githubusercontent.com/5960563/197624834-0b9cd893-fee8-428b-88cb-d0aff54004a0.png)

Then you can just delete those lines and use `self.width()` and `self.height()` for the calculation of posX and posY.

```suggestion
        width = self.width()
        height = self.height()
```